CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [18.1.0](https://github.com/theforeman/puppet-foreman_proxy/tree/18.1.0) (2021-04-29)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy/compare/18.0.0...18.1.0)
+
+**Implemented enhancements:**
+
+- Support Ubuntu 20.04 [\#669](https://github.com/theforeman/puppet-foreman_proxy/pull/669) ([ekohl](https://github.com/ekohl))
+- don't manage runner repo on Debian [\#664](https://github.com/theforeman/puppet-foreman_proxy/pull/664) ([evgeni](https://github.com/evgeni))
+
+**Fixed bugs:**
+
+- Allow puppet-puppet \< 16.0.0 [\#665](https://github.com/theforeman/puppet-foreman_proxy/pull/665) ([wbclark](https://github.com/wbclark))
```

Comment:
Feels more like an enhancement to me. Doesn't matter that much since we're doing a minor version bump anyway.